src/MOIWrapper.jl

```diff
@@ -400,5 +400,11 @@ function MOI.get(cbc_optimizer::Optimizer, object::MOI.PrimalStatus)
         return MOI.FeasiblePoint
     elseif isProvenInfeasible(cbc_optimizer.inner)
         return MOI.InfeasiblePoint
+    else
+        return MOI.NoSolution
     end
 end
+
+function MOI.get(cbc_optimizer::Optimizer, ::MOI.DualStatus)
```

Comment:
No need to name the first argument

Reply:
The code is riddled with things like this, so better to just address all at once. See #88